lib/rubocop/cop/gemspec/ruby_version_globals_usage.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gemspec
+      # Checks that `RUBY_VERSION` constant is not used in gemspec
+      #
+      # @example
+      #   # When `RUBY_VERSION` is used in gemspec
```

Comment:
I think that this comment can be deleted because it is redundant with the cop explanation.